packages/lavapack/src/runtime-cjs.js

```diff
@@ -2,16 +2,7 @@
 
 ;(function() {
 
-  // identify the globalRef
-  const globalRef = (typeof globalThis !== 'undefined') ? globalThis : (typeof self !== 'undefined') ? self : (typeof global !== 'undefined') ? global : undefined
-  if (!globalRef) {
-    throw new Error('Lavamoat - unable to identify globalRef')
-  }
-
-  // polyfill globalThis
-  if (globalRef && !globalRef.globalThis) {
-    globalRef.globalThis = globalRef
-  }
```

Comment:
@legobeat these files were not covered by your PR. Was that deliberate?
We probably can assume devs (who use this runtime) would not need a deprecation warning, but I want to make sure that's what you meant.

Reply:
It was indeed not deliberate! #461